src/assertions/OrderAssertions.ts

```diff
@@ -56,38 +58,48 @@ export class OrderAssertions {
     this.setTokenAssertions = new SetTokenAssertions(web3);
   }
 
+  public setCoreAddress(coreAddress: Address) {
```

Comment:
This pattern is error prone. Pass into each assertion when needed or make it required in the constructor. Something we do in [contracts](https://github.com/SetProtocol/set-protocol-contracts/blob/master/utils/coreWrapper.ts#L143) is we keep a pointer to an instance of core so it doesn't need to be created every time.